src/main/java/com/cloudant/client/org/lightcouch/Changes.java

```diff
@@ -215,7 +212,6 @@ private void setNextRow(Row nextRow) {
     }
 
     private void terminate() {
-        httpGet.abort();
```

Comment:
I did a bit of checking and closing the `InputStream` should be sufficient for this, so I think it is ok.